nautobot/dcim/migrations/0059_alter_cable_status_alter_consoleport__path_and_more.py

```diff
@@ -66,6 +66,20 @@ class Migration(migrations.Migration):
                 on_delete=django.db.models.deletion.CASCADE, to="dcim.devicetype"
```

Comment:
Were these just the updates that came in to 2.2.0 after the beta?

Reply:
These files are part of the [generated migrations](https://docs.djangoproject.com/en/4.0/releases/4.0/#migrations-autodetector-changes) that were autogenerated when migrating to Django 4. I'd generated one version of them when I first created the prototype branch a few months back, then this week I updated to latest `next` and addressed some resulting merge conflicts and necessary renumbering, but discovered in this PR's CI that there were additional model changes that needed to be accounted for as well.

The full text of these migrations is in the base `prototype/3581-django-4.2` branch.

Hope that makes sense?

Reply:
Yeah that makes sense. Hopefully the changes they made in 4.0 fix the issues we see where model state is broken when reverting migrations.